packages/account/src/providers/fuel-graphql-subscriber.ts

```diff
@@ -48,10 +50,8 @@ export class FuelGraphqlSubscriber implements AsyncIterator<unknown> {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const { data, errors } = this.events.shift()!;
         if (Array.isArray(errors)) {
-          throw new FuelError(
-            FuelError.CODES.INVALID_REQUEST,
-            errors.map((err) => err.message).join('\n\n')
-          );
+          const errorMessage = errors.map((err) => err.message).join('\n\n');
+          handleGqlErrorMessage(errorMessage);
         }
```

Comment:
I'd of personally, of create a `assertNoGqlErrors` which takes the whole errors array. Then parse the individual errors from there and throw if required.

Currently, as it stands, if there were two errors then the detection in `handleGqlErrorMessage` would fall over.